generator/generator_tests/feature_builder_test.cpp

```diff
@@ -218,6 +220,54 @@ UNIT_CLASS_TEST(TestWithClassificator, FBuilder_ParamsParsing)
   TEST_EQUAL(params.house.Get(), "0", ());
 }
 
+UNIT_CLASS_TEST(TestWithClassificator, FBuilder_Housenumbers)
+{
+  FeatureBuilderParams params;
+  std::string_view name;
+
+  params.MakeZero();
+  name = {};
+  params.AddHouseName("75a"); // addr:housenumber
+  params.AddHouseName("Sandpiper"); // addr:housename
+  TEST_EQUAL(params.house.Get(), "75a", ());
+  TEST(params.name.GetString(StringUtf8Multilang::kDefaultCode, name), ());
+  TEST_EQUAL(name, "Sandpiper", ());
+
+  params.MakeZero();
+  name = {};
+  params.AddHouseName("75"); // addr:housenumber
```

Comment:
Should we have `AddHouseNumber("75")` here?

Reply:
both housenumbers and housenames are added via AddHouseName() always in the generator